sdk/ml/azure-ai-ml/tests/dsl/unittests/test_dsl_group.py

```diff
@@ -326,15 +326,43 @@ def str_param_pipeline(str_param, group: ParamClass):
         pipeline_job.inputs.str_param = ParamClass()
         # Note: this is the expected behavior.
         assert (
-            pipeline_job._to_dict()["inputs"]["str_param"]
-            == "TestDSLGroup.test_assign_group_invalid.<locals>.ParamClass(str_param='string_by_default')"
+                pipeline_job._to_dict()["inputs"]["str_param"]
+                == "TestDSLGroup.test_assign_group_invalid.<locals>.ParamClass(str_param='string_by_default')"
         )
 
         pipeline_job.inputs.str_param = "test"
         with pytest.raises(Exception) as e:
             pipeline_job.inputs.group = "test"
         assert "'group' is expected to be a parameter group, but got <class 'str'>." in str(e)
 
+    def test_group_outputs(self):
+        @group
```

Comment:
add some success case that with prmitive type annotations, .e.g.  str, float..

Reply:
Added primitive type cases, they can be created as group object, but cannot be used in @pipeline